doc/manual/core_device.rst

```diff
@@ -71,19 +71,21 @@ With the CLOCK hardware, the TTL lines are mapped as follows:
 
 The board has RTIO SPI buses mapped as follows:
 
-+--------------+-------------+-------------+-----------+------------+
-| RTIO channel | CS_N        | MOSI        | MISO      | CLK        |
-+==============+=============+=============+===========+============+
-| 22           | AMS101_CS_N | AMS101_MOSI |           | AMS101_CLK |
-+--------------+-------------+-------------+-----------+------------+
-| 23           | SPI0_CS_N   | SPI0_MOSI   | SPI0_MISO | SPI0_CLK   |
-+--------------+-------------+-------------+-----------+------------+
-| 24           | SPI1_CS_N   | SPI1_MOSI   | SPI1_MISO | SPI1_CLK   |
-+--------------+-------------+-------------+-----------+------------+
-| 25           | SPI2_CS_N   | SPI2_MOSI   | SPI2_MISO | SPI2_CLK   |
-+--------------+-------------+-------------+-----------+------------+
-
-The DDS bus is on channel 26.
++--------------+--------------+--------------+--------------+------------+
+| RTIO channel | CS_N         | MOSI         | MISO         | CLK        |
++==============+==============+==============+==============+============+
+| 22           | AMS101_CS_N  | AMS101_MOSI  |              | AMS101_CLK |
```

Comment:
You added your SPI PHY *after* those SPI buses, so their channel numbers are not affected. Only the DDS channel number is affected, as far as I can tell (off the top of my head).